scripts/gitlab/build_and_test.sh

```diff
@@ -123,7 +123,7 @@ then
     echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 
     # Map CPU core allocations
-    declare -A core_counts=(["lassen"]=40 ["ruby"]=28 ["corona"]=32)
+    declare -A core_counts=(["lassen"]=40 ["ruby"]=28 ["corona"]=32 ["rzansel"]=48)
```

Comment:
Why this change? We don't do any Gitlab testing on RZ machines?

Reply:
I was debugging failures I was seeing at some point and wanted to reproduce them on ansel, this script doesn't work unless you define this for the given hostname.